app/bundles/FormBundle/Entity/Form.php

```diff
@@ -546,6 +546,29 @@ public function getFieldAliases()
     }
 
     /**
+     * Loops through the form fields and returns an array of fields with mapped data.
+     *
+     * @return array<int, array<string, int|string>>
+     */
+    public function getMappedFieldValues(): array
+    {
+        return array_filter(
+            array_map(
+                function (Field $field) {
+                    return [
+                        'idFormField'  => $field->getId(),
```

Comment:
Do you think the key could be changed to
```suggestion
                        'formFieldId'  => $field->getId(),
```
As that's how the ID fields are called in Mautic and it more resembles English.